imi/src/main/resources/imi.properties

```diff
@@ -17,6 +17,14 @@ imi.max_query_block=1000
 #4.3.1 TARGET_FILE Notification API
 imi.target_file_notification_url=http://localhost/obdmanager/notifytargetfile
 
+#url to use to notify the IVR system that the CDR file they sent was processed
+#4.3.2 CDRFileProcessedStatus Notification API
+imi.cdr_file_notification_url=http://localhost/obdmanager/NotifyCDRFileProcessedStatus
+
 #4.4.1 Target File Format
 #field #2 ServiceId - Unique Id provided by IMImobile for a particular service
-imi.target_file_imi_service_id=imiserviceid
\ No newline at end of file
+imi.target_file_imi_service_id=imiserviceid
```

Comment:
I thought the serviceId was going to change depending on the retry count for each record.  So fresh base calls would get serviceId 1.  First retry would get serviceId 2 and second retry some other service Id.  They would then do prioritization on their end based on the service id.